test/Microsoft.Azure.SignalR.Emulator.Tests/Controllers/SignalRServiceEmulatorWebApiTests.cs

```diff
@@ -0,0 +1,423 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Azure.SignalR.Emulator.Controllers;
+using Microsoft.Azure.SignalR.Emulator.HubEmulator;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.SignalR.Emulator.Tests.Controllers
+{
+    public class SignalRServiceEmulatorWebApiTests
+    {
+        private readonly Mock<IDynamicHubContextStore> storeMock;
+        private readonly Mock<ILogger<SignalRServiceEmulatorWebApi>> loggerMock;
+        private readonly SignalRServiceEmulatorWebApi controller;
+
+        // Constants
+        private const string testHub = "testHub";
+        private const string testConnectionId = "testConnectionId";
+        private const string testGroup = "testGroup";
+        private const string testUser = "testUser";
+        private const string testApplication = "testApplication";
+
+        private const string MicrosoftErrorCode = "x-ms-error-code";
+
+        private const string Warning_Connection_NotExisted = "Warning.Connection.NotExisted";
+        private const string Warning_Group_NotExisted = "Warning.Group.NotExisted";
+        private const string Warning_User_NotExisted = "Warning.User.NotExisted";
+        private const string Error_Connection_NotExisted = "Error.Connection.NotExisted";
+        private const string Info_User_NotInGroup = "Info.User.NotInGroup";
+
+        public SignalRServiceEmulatorWebApiTests()
+        {
+            storeMock = new Mock<IDynamicHubContextStore>();
+            loggerMock = new Mock<ILogger<SignalRServiceEmulatorWebApi>>();
+            controller = new SignalRServiceEmulatorWebApi(storeMock.Object, loggerMock.Object);
+
+            HttpContext httpContext = new DefaultHttpContext();
+            var controllerContext = new ControllerContext()
+            {
+                HttpContext = httpContext,
+            };
+
+            controller.ControllerContext = controllerContext;
+        }
+
+        // CheckConnectionExistence Tests
+        [Fact]
+        public void CheckConnectionExistenceValidConnectionReturnsOk()
+        {
+            // Arrange
+            var connectionContext = new DefaultConnectionContext();
+            connectionContext.ConnectionId = testConnectionId;
+
+            var mockLoggerFactory = new Mock<ILoggerFactory>();
```

Comment:
Just a hint. If the logger/loggerFactory is not used in test, you can also use `NullLoggerFactory.Instance` to simplify.